### DISCUSSION TITLE
二次封装v5的组件，样式如何修改才能接受ConfigProvider的控制，还有如何提供自定义token？

### DISCUSSION THREAD

**Discussion by qc-z** [ant-design/ant-design#40702] [General]
### What problem does this feature solve?

希望能提供一个具体样式方案（怎么写样式？）来实现二次封装v5的组件

### What does the proposed API look like?

可以在ConfigProvider提供antd没有的token进去

**Comment by MadCcc** [comment]
```tsx
import { theme } from 'antd';

const useMyToken = () => {
  const { token } = theme.useToken();
  return {
    ...token,
    customToken: 'custom',
  }; 
};
```

**Comment by arvinxx** [comment]
> 希望能提供一个具体样式方案（怎么写样式？）来实现二次封装v5的组件

https://ant-design-antd-style-preview-pr-19.surge.sh/guide/create-styles

> 可以在ConfigProvider提供antd没有的token进去

参看： https://ant-design-antd-style-preview-pr-19.surge.sh/guide/components-usage
